src/uikit/Multicard/index.js

```diff
@@ -1,24 +1,24 @@
-import React, { Component, createContext } from 'react';
+import React, { Component } from 'react';
 
 import CardHeader from 'uikit/Card/CardHeader';
 import CardContent from 'uikit/Card/CardContent';
 import { CardWrapper, HeaderWrapper } from 'uikit/Card/styles';
-import posed, { PoseGroup } from 'react-pose';
+// import posed from 'react-pose';
 import LoadingSpinner from 'uikit/LoadingSpinner';
 import TabMenu from './TabMenu';
 import IndexDots from './IndexDots';
 
-const AnimatedChild = posed.div({
-  enter: {
-    y: 0,
-    opacity: 1,
-  },
-  exit: {
-    y: 50,
-    opacity: 0,
-    transition: { duration: 150 },
-  },
-});
+// const AnimatedChild = posed.div({
```

Comment:
Remove instead of comment out?

Reply:
I think Ciaran wanted these around so he can revisit to add some animation :P I'm just silencing the linters